docs/source/developer/contributing.rst

```diff
@@ -461,16 +462,16 @@ Main documentation
 ''''''''''''''''''
 
 This documentation is under the ``docs`` directory of the `Numba repository`_.
-It is built with `Sphinx <http://sphinx-doc.org/>`_ and
-`numpydoc <https://numpydoc.readthedocs.io/>`_, which are available using
-conda or pip; i.e. ``conda install sphinx numpydoc``.
+It is built with `Sphinx <http://sphinx-doc.org/>`_, `numpydoc
+<https://numpydoc.readthedocs.io/>`_ and the `sphinx-rtd-theme
+<https://sphinx-rtd-theme.readthedocs.io/en/stable/>`.
```

Comment:
I think this needs a following underscore to render as a link:

```suggestion
<https://sphinx-rtd-theme.readthedocs.io/en/stable/>_`.
```

Reply:
<img width="1124" alt="Screen Shot 2023-05-04 at 10 24 17" src="https://user-images.githubusercontent.com/41563/236149951-e082e4ac-ba45-4f00-913e-8142c297826c.png">
 
this one also

Reply:
The underscore must come after the backtick. I fixed this in 4a030ee85f716e5b2c523ff2f02a57a27560a79b